lib/peep.rb

```diff
@@ -0,0 +1,35 @@
+require 'pg'
+
+class Peep
+
+    attr_reader :id, :message, :date, :time
+
+    def initialize(id:, message:, date:, time:)
+        @id = id
+        @message = message
+        @date = date
+        @time = time
+    end
+
+    def self.all
+        if ENV['ENVIRONMENT'] == 'test'
+            connection = PG.connect(dbname: 'chitter_test')
+          else
+            connection = PG.connect(dbname: 'chitter')
+          end
+        result = connection.exec('SELECT * FROM peeps;')
+        result.map do |peep|
+            Peep.new(id: peep['id'], message: peep['message'], date: peep['date'], time: peep['time'])
+        end
+    end
+
+    def self.post(message:)
```

Comment:
Maybe .create would be a better variable name, as .post seems very similar to the function of .all